test/game/validator_test.exs

```diff
@@ -83,5 +84,23 @@ defmodule ValidatorTest do
 
       assert Validator.validate_setup(user_input, computer_types) == {:ok, HardComputerPlayer}
     end
+
+    test "returns an invalid error when the user enters a number other than 1 or 2 during the player selection setup" do
+      user_input = "4"
+
+      computer_types = [%{type: EasyComputerPlayer, name: "Easy"}, %{type: HardComputerPlayer, name: "Unbeatable"}]
```

Comment:
Nitpick: what do you think about applying a little formatting to make these more legible? For example:
```ex
computer_types = [
  %{type: EasyComputerPlayer, name: "Easy"},
  %{type: HardComputerPlayer, name: "Unbeatable"}
]
...
```

Reply:
I like this alot and would say I am in the same camp of being in favor of anything that makes the code legible! 😄